content/python/concepts/statsmodels/statsmodels.md

```diff
@@ -0,0 +1,146 @@
+---
+Title: 'Statsmodels'
+Description: 'The statsmodels library is used to estimate statistical models and perform statistical tests in Python.' 
+Subjects:
+  - 'Data Science'
+  - 'Statistics'
```

Comment:
`Statistics` is not a valid subject. Please refer to the list of subjects in [subjects.md](https://github.com/Codecademy/docs/blob/main/documentation/subjects.md).

```suggestion
  - 'Data Visualization'
  - 'Computer Science'
```